src/main/java/mate/controller/DeleteManufacturerController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.dao.ManufacturerDao;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+public class DeleteManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerDao dao =
+            (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
Please rework method to work via POST method, get should be used for obtaining information, not modification